README.md

```diff
@@ -1,57 +1,4 @@
-# Northcoders News API
+# How to create the environment variables to clone this project and run it locally.
 
-## Background
+You will need to create two .env files for your project: .env.test and .env.development. Into each, add PGDATABASE=<database_nc_news>, PGDATABASE=<database_nc_news_test>, which is the correct database name for that environment. Double check that these .env files are .gitignored.
 
```

Comment:
Please be more specific about which file will need which PGDATABASE line. Also where do they make the .env files?

It would also be useful to tell the user to run 'npm install' so they have all the relevant dependencies.